docs/extensibility/walkthrough-using-a-shortcut-key-with-an-editor-extension.md

```diff
@@ -41,8 +41,21 @@ You can respond to shortcut keys in your editor extension. The following walkthr
 ```csharp  
 this.layer = view.GetAdornmentLayer("PurpleCornerBox");  
 ```  
+
+In the KeyBindingTestTextViewCreationListener.cs class file, change the name of the AdornmentLayer from **KeyBindingTest** to **PurpleCornerBox**:
   
-## Defining the Command Filter  
+    ```csharp  
+    [Export(typeof(AdornmentLayerDefinition))]  
+    [Name("PurpleCornerBox")]  
+    [Order(After = PredefinedAdornmentLayers.Selection, Before = PredefinedAdornmentLayers.Text)]  
+    public AdornmentLayerDefinition editorAdornmentLayer;  
+    ```  
+
+## Handling TYPECHAR Command
+Prior to Visual Studio 2017 version 15.6 the only way to handle commands in an editor extention was implementing an <xref:Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget> based command filter. Visual Studio 2017 version 15.6 introduced a modern simplified approach based on editor command handlers. The next two sections demonstarte how to handle a command using both legacy and modern approach.
```

Comment:
typos: extention -> extension, demonstarte -> demonstrate
"both the legacy and modern approach"